kwok/Dockerfile

```diff
@@ -4,11 +4,12 @@ ARG CLUSTER_NAME=kwok
 ENV CLUSTER_NAME=${CLUSTER_NAME}
 
 COPY --chmod=755 user-signups.sh entrypoint.sh toolchain.dev.openshift.com_usersignups.yaml ./
+COPY --chmod=644 inittab /etc/inittab
 
 RUN KWOK_KUBE_APISERVER_PORT=0 kwokctl create cluster --name "$CLUSTER_NAME" || exit 1 && \
     kwokctl --name="$CLUSTER_NAME" kubectl apply -f toolchain.dev.openshift.com_usersignups.yaml && \
     sh ./user-signups.sh && \
     rm user-signups.sh toolchain.dev.openshift.com_usersignups.yaml && \
     kwokctl stop cluster --name="$CLUSTER_NAME"
 
-ENTRYPOINT ["/entrypoint.sh"]
+ENTRYPOINT ["/sbin/init"]
```

Comment:
If I understand correctly, the `ENTRYPOINT ["/entrypoint.sh"]` part is misleading. We're not running our `entrypoint.sh`, but the [one coming from the base image](https://github.com/kubernetes-sigs/kwok/blob/main/images/cluster/entrypoint.sh).

With this change, are we still starting it?

Reply:
(or still running something that does a similar job)

Reply:
yes @yftacherzog .

Reply:
no, we are overriding the one in the base image with our own during the image build. We have no need for the functionality in the base image entry point which has to do with setting up a cluster which we do during the build.

Reply:
Only change why it couldn't reload inittab might be on line 15. https://github.com/srivickynesh/segment-bridge/pull/1/files

Reply:
I don't think our `entrypoint.sh` is being copied to the image, so the existing functionality is not using those lines that this PR is putting in `kwok/inittab`.

Reply:
@srivickynesh you are not running init at all in that patch, and you've added a lot of unnecessary stuff that probably causes issues such as command for starting TTYs.

Reply:
those are all the defaults that comes within inittab

Reply:
ok. I realize now that the line of copying the entrypoint was not a part of the code I was dealing with yesterday.

Reply:
@srivickynesh those defaults are for booting whole servers, you are not doing that here...